guidance/models/transformers/_transformers.py

```diff
@@ -174,6 +76,203 @@ def __init__(
             transformers_tokenizer.eos_token_id,
         )
 
+    def _tokenizer(self, model: str, **kwargs) -> Union[
+        "transformers_package.PreTrainedTokenizer",
+        "transformers_package.PreTrainedTokenizerFast",
+    ]:
+        # make sure transformers is installed
+        if not has_transformers:
+            raise Exception("Please install transformers with `pip install transformers`")
+
+        try:
+            tokenizer = transformers_package.AutoTokenizer.from_pretrained(
+                model, use_fast=False, **kwargs
+            )
```

Comment:
Previously, we asserted `_byte_decoder_has_all_bytes` here, potentially causing a fallback to `fast`. I instead moved that check into `_byte_tokens`.

Note: we could call `_byte_tokens` inside of this try/except instead of further up the stack if we want to fall-back to `fast` on failure... I'm not sure what semantics are better...